relayer-cli/src/commands/tx/client.rs

```diff
@@ -214,18 +215,26 @@ pub struct TxUpgradeClientCmd {
         help = "Identifier of the client to be upgraded"
     )]
     client_id: ClientId,
+
+    #[clap(
+        long = "upgrade-height",
+        required = true,
+        value_name = "HOST_TARGET_UPGRADE_HEIGHT",
+        help = "The height at which the host chain should halt in order to perform the client upgrade"
```

Comment:
This should be `"The height at which the reference chain halts for the upgrade"`